pkg/ccl/backupccl/testdata/backup-restore/alter-schedule

```diff
@@ -0,0 +1,188 @@
+new-server name=s1 allow-implicit-access
+----
+
+# Create test schedules.
+
+exec-sql
+create schedule datatest for backup into 'nodelocal://1/example-schedule' recurring '@daily' full backup '@weekly';
+----
+
+let $fullID $incID
+with schedules as (show schedules) select id from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+
+query-sql
+with schedules as (show schedules) select id, state, recurrence, owner, command from schedules where label='datatest' order by command->>'backup_type' asc;
+----
+$fullID <nil> @weekly root {"backup_statement": "BACKUP INTO 'nodelocal://1/example-schedule' WITH detached", "chain_protected_timestamp_records": true, "dependent_schedule_id": $incID, "unpause_on_success": $incID}
+$incID Waiting for initial backup to complete @daily root {"backup_statement": "BACKUP INTO LATEST IN 'nodelocal://1/example-schedule' WITH detached", "backup_type": 1, "chain_protected_timestamp_records": true, "dependent_schedule_id": $fullID}
+
+# Disable incremental backup.
+
+exec-sql
+alter backup schedule $fullID set full backup always;
```

Comment:
Is there a test that both sets full backup always and changes the recurrence (eg. to daily)?

Reply:
there is now!